src/udev/udevd.c

```diff
@@ -128,6 +128,7 @@ typedef struct Event {
 
         uint64_t seqnum;
         uint64_t blocker_seqnum;
+        usec_t wait_until;
```

Comment:
hmm, maybe call this "retry_again_timeout_usec" or so? i.e. emphasize that this is about retry again?

Reply:
so, with this logic we'll still retry idefinitely, no?

I think we should have also a time at which we will give up, and fail the event. So that we actually get the guarantee events are dropped eventually from the queue.

Hence, maybe a logic: first time a worker sends EVENT_RESULT_TRY_AGAIN we'll set two times: 
1. `retry_again_next_usec` to a short time (1s or so)
2. `retry_again_timeout_usec` to a long time (3 min or so)

And then, after each retry bump the "next" but not the "timeout", which we'll fail instead.